tests/test_documents.py

```diff
@@ -161,3 +166,101 @@ async def test_add_document_ingest_error(client):
         client=client, collection=collection.name, document_id=document.id
     )
     assert len(chunks) == 0
+
+async def test_upload_document_content_invalid(client, doc_fixture):
+    with open(NEARLY_EMPTY_PATH, 'rb') as file:
+        response = await upload_document_content.asyncio_detailed(
+            client=client,
+            document_id=1_000_000,
+            body=BodyUploadDocumentContent(
+                content=File(
+                    payload=file,
+                    file_name="file-name-1",
+                    mime_type="application/pdf",
+                ),
+            ),
+        )
+    assert response.status_code == 404
+
+async def test_document_lifecycle(client, doc_fixture):
```

Comment:
Maybe "test_update_document_content" or something? Eg., there are a lot of different lifecycles (eg., changing the URL) that aren't part of this.